man/dnf.plugin.kpatch.8

```diff
@@ -0,0 +1,34 @@
+.TH man 8 "23 Apr 2020" "1.0" "dnf-kpatch man page"
```

Comment:
File license/copyright information is missing.

Reply:
Is the license added at the bottom of the file enough?

Reply:
It depends; generally, man pages in some projects may have different license (CC-like, for example), and some projects insist on having license (or SPDX tag) in each file. If kpatch is not one of those, then one may assume that the project's license applies to the man page as well, yes.

Reply:
I think the project's license applies to the man pages. @jpoimboe would you confirm that? Or is there someone else to discuss this?

Reply:
Yes, the project license applies to everything, including the man pages.